[2025-10-31T02:52:02.832+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-31T02:52:02.860+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_postgres_dag.load_to_postgres manual__2025-10-31T02:36:55.515416+00:00 [queued]>
[2025-10-31T02:52:02.868+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_postgres_dag.load_to_postgres manual__2025-10-31T02:36:55.515416+00:00 [queued]>
[2025-10-31T02:52:02.869+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2025-10-31T02:52:02.881+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2025-10-31 02:36:55.515416+00:00
[2025-10-31T02:52:02.887+0000] {standard_task_runner.py:63} INFO - Started process 351 to run task
[2025-10-31T02:52:02.890+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'api_to_postgres_dag', 'load_to_postgres', 'manual__2025-10-31T02:36:55.515416+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/api_to_postgres_dag.py', '--cfg-path', '/tmp/tmpyeb9pvtb']
[2025-10-31T02:52:02.892+0000] {standard_task_runner.py:91} INFO - Job 71: Subtask load_to_postgres
[2025-10-31T02:52:02.945+0000] {task_command.py:426} INFO - Running <TaskInstance: api_to_postgres_dag.load_to_postgres manual__2025-10-31T02:36:55.515416+00:00 [running]> on host 754154eb0275
[2025-10-31T02:52:03.026+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='lenny' AIRFLOW_CTX_DAG_ID='api_to_postgres_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-10-31T02:36:55.515416+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-31T02:36:55.515416+00:00'
[2025-10-31T02:52:03.028+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-31T02:52:03.055+0000] {logging_mixin.py:188} INFO - âœ… Loaded data into PostgreSQL table.
[2025-10-31T02:52:03.056+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-31T02:52:03.056+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-31T02:52:03.066+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=api_to_postgres_dag, task_id=load_to_postgres, run_id=manual__2025-10-31T02:36:55.515416+00:00, execution_date=20251031T023655, start_date=20251031T025202, end_date=20251031T025203
[2025-10-31T02:52:03.103+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-31T02:52:03.123+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-31T02:52:03.125+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
